.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,21 @@
+---
+name: '[INTERNAL] Release'
+about: Release a version
+title: 'Release Version v$version'
+labels: ''
+assignees: ''
+
+---
+
+# Release Workflow
+
+- [ ] Check that the `.mailmap` and `AUTHORS` files are up to date
+- [ ] Update `ICINGA2_VERSION`
+- [ ] Update `CHANGELOG.md`
+- [ ] Create and push a signed tag for the version
+- [ ] Build and release DEB and RPM packages
+- [ ] Update bundled Windows dependencies
```

Comment:
Windows dependency update must happen before tagging the release.

Reply:
True. Fixed.